vanhouse/src/style.css

```diff
@@ -0,0 +1,52 @@
+#table{

```

Comment:
If you have the newest project_3, you'd see we have a `styles` folder. You might want to make a css file for each component you want to style and put styles there.